config/routes.rb

```diff
@@ -19,6 +19,7 @@
       namespace :users do
         get '/by_name', to: 'by_name#index'
         get '/by_cohort', to: 'by_cohort#index'
+        get '/find_by_github', to: 'find_by_github#show'
```

Comment:
@wlffann Maybe yea. Those are ally `find_all` and this one is just `find` so I wasn't sure how to try to differentiate that.